sdk-wasm/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "kepler-sdk-wasm"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Probably should be 0.2.0 as the move to siwe-recap introduces breaking changes in the output